app/src/main/java/org/fossasia/activities/MainActivity.java

```diff
@@ -268,6 +268,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
+    private void startDownloadSchedule() {
```

Comment:
Calling this function will again start the splash activity (starting activity of the app), refreshing should be handled in background thread.